docs-examples/example-java/src/test/java/io/micronaut/rabbitmq/docs/headers/ProductClient.java

```diff
@@ -23,5 +24,9 @@
     void send(@Header String productSize, // <3>
               @Header("x-product-count") Long count,
               byte[] data);
+
+    @Binding("product")
+    void send(RabbitHeaders rabbitHeaders, // <4>
```

Comment:
Same as above...

Reply:
Changes to the  MessageHeaders interface.